src/main/java/com/google/codeu/data/Datastore.java

```diff
@@ -94,7 +94,7 @@ public void storeUser(User user) {
   public User getUser(String email) {
 
     Query query = new Query("User")
-      .setFilter(new Query.FilterPredicate("email", FilterOperator.EQUAL, email));
+    .setFilter(new Query.FilterPredicate("email", FilterOperator.EQUAL, email));
```

Comment:
I would try to avoid changing whitespace like this, since it makes it harder to know exactly what you changed in this PR.

Reply:
sorry that was on accident.